### PR TITLE
[SELS-TASK] Create admin view user list markup

### PR DESCRIPTION
**Pull request by jstephend-sun**
subbranch of PR 14

Asana [Task](https://app.asana.com/0/1201478050789792/1201478741328068/f)
Heroku [link](jstephend-els-fe.herokuapp.com)

notes:
some of the file changes are just import refactoring, sorry for the number of file changes

commands
```
npm i
npm start
```

expected output:
can see user list form (admin)

screenshots:
image

### CODE REVIEW

File: frontend/src/components/AdminUser/AdminUserBody.tsx [hunk 1]
```diff
@@ -0,0 +1,44 @@
...
+            />
+          </TableBodyRowData>
+          <AdminUserAction user={user} />
+        </TableBodyRow>
+      );
+    });
+  }
+
+  return <>{renderUsers()}</>;
+};
```
Comment (kimjo77) [inline frontend/src/components/AdminUser/AdminUserBody.tsx]: I just realize this is a bit unnecessarily decoupled. Unless you have plans to reuse renderUsers() function i think you wouldnt need to extract it to a function pa you can simply have it like so:
```js
export const AdminUserBody = ({ users }: AdminUserBodyProps): JSX.Element => {

  return (
    <>
      { users?.map((user) => {
        return (
          <TableBodyRow key={user.id}>
            <TableBodyRowData>
              <div className="flex items-center">
                <TableBodyRowAvatar src={user.avatar} />
                <div className="ml-4">{user.name}</div>
              </div>
            </TableBodyRowData>
            <TableBodyRowData>{user.email}</TableBodyRowData>
            <TableBodyRowData>
              <TableBodyRowStatus
                text={user.is_admin ? 'Admin' : 'User'}
                statusBg={user.is_admin ? 'green-100' : 'cyan-100'}
                statusText={user.is_admin ? 'green-900' : 'cyan-900'}
              />
            </TableBodyRowData>
            <AdminUserAction user={user} />
          </TableBodyRow>
        );
      })}
    </>
  );
};
```

File: frontend/src/components/AdminQuiz/AdminQuizBody.tsx [hunk 2]
```diff
@@ -23,7 +19,7 @@ const AdminQuizBody = ({ records }: AdminQuizBodyProps): JSX.Element => {
     });
   }
 
-  return <TableBody>{renderList()}</TableBody>;
+  return <>{renderList()}</>;
```
Comment (kimjo77) [inline frontend/src/components/AdminQuiz/AdminQuizBody.tsx]: ahh also this page

File: frontend/src/components/AdminQuiz/AdminQuizBody.tsx [hunk 3]
```diff
@@ -1,16 +1,12 @@
...
-import { Quiz } from './types';
+import { Quiz, AdminQuizAction } from '.';
+import { TableBodyRow, TableBodyRowData, TableBody } from '../Table';
 
 interface AdminQuizBodyProps {
   records: Quiz[];
 }
 
-const AdminQuizBody = ({ records }: AdminQuizBodyProps): JSX.Element => {
+export const AdminQuizBody = ({ records }: AdminQuizBodyProps): JSX.Element => {
```
Comment (kimjo77) [inline frontend/src/components/AdminQuiz/AdminQuizBody.tsx:9]: also i notice you export in this section but at the very bottom you still called export default, is there a reason for this?

Comment (jstephend-sun) [reply frontend/src/components/AdminQuiz/AdminQuizBody.tsx:9]: my thinking is that devs can import from the index as { AdminQuizBody }
or import from the file as just AdminQuizBody

File: frontend/src/components/AdminUser/AdminUserBody.tsx (hunk 1 above)
Comment (jstephend-sun) [reply frontend/src/components/AdminUser/AdminUserBody.tsx]: changed to suggestion

File: frontend/src/components/AdminQuiz/AdminQuizBody.tsx (hunk 2 above)
Comment (jstephend-sun) [reply frontend/src/components/AdminQuiz/AdminQuizBody.tsx]: changed to suggestion

File: frontend/src/components/AdminQuiz/AdminQuizBody.tsx (hunk 3 above)
Comment (kimjo77) [reply frontend/src/components/AdminQuiz/AdminQuizBody.tsx:9]: I'm not really sure about this approach yet though I will let you experiment on it.

**Review by kimjo77 — APPROVED** [review]

**Pull request merged**